### PR TITLE
IN-35 feat: add sidebar and signOut

### PR DESCRIPTION
**Pull request by web-serge**

### CODE REVIEW

File: src/components/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      open={isLogout}
+      trigger={
+        <Button>
+          <IconLogOut /> {t.sidebar.logOut}
+        </Button>
+      }
+    >
+      <div className={s.modalBody}>
+        <Typography variant={"regular_text_16"}>
+          {`${t.sidebar.logOutModalMessage} ${meData?.email}?`}
```
Comment (Abrosimov96) [inline src/components/sidebar/sidebar.tsx:97]: You can make a function inside locales and use it {t.sidebar.logOutModalMessage(email)}

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -25,10 +28,19 @@ export default function App({ Component, ...rest }: AppProps) {
...
+
+  if (isLoading) {
+    return <h1>Loading...</h1>
+  }
+
   return (
     <>
-      <Header />
-      <main>{children}</main>
+      <Header isMe={isError} />
```
Comment (Abrosimov96) [inline src/pages/_app.tsx:39]: isLoggedIn={!isError} may be it's better name for this variable

**Review by Abrosimov96 — APPROVED** [review]

File: src/pages/_app.tsx (hunk 2 above)
Comment (vitaliiRss) [reply src/pages/_app.tsx:36]: fixed

**Review by SergFediaev — APPROVED** [review]

File: src/pages/_app.tsx (hunk 2 above)
Comment (vitaliiRss) [reply src/pages/_app.tsx:36]: Changed to `isSignedIn={isError}`

File: src/components/sidebar/sidebar.tsx (hunk 1 above)
Comment (vitaliiRss) [reply src/components/sidebar/sidebar.tsx:97]: fixed `{t.sidebar.logOutModalMessage(dataMe?.email ?? "")}`

**Pull request merged**